crates/parser/src/grammar/generic_args.rs

```diff
@@ -102,19 +98,25 @@ pub(super) fn const_arg(p: &mut Parser) {
             p.bump(T![-]);
             expressions::literal(p);
             lm.complete(p, PREFIX_EXPR);
-            m.complete(p, CONST_ARG);
         }
         // test const_arg_path
-        // struct S<const N: u32 = u32::MAX>;
+        // type T = S<u32::MAX>;
```

Comment:
This should stay unchanged, otherwise this test won't hit this path(as it'll be parsed as a non-const generic argument).

Reply:
You're right that it's wrong, I misread the newly generated rast. But reverting it isn't quite right either - it was testing param lists instead of arg lists, i.e. it was generating
```
GENERIC_PARAM_LIST
  CONST_PARAM
    CONST_ARG
      PATH_EXPR
```
but it should be aiming to generate
```
GENERIC_ARG_LIST
  CONST_ARG
    PATH_EXPR
```
I can add the old test to generic_params.rs

Reply:
Right, we probably won't hit the code in generc_args at all here, if there is no such test in generic_params yet we can add it there, otherwise just remove it.